users/contrib.rst

```diff
@@ -88,33 +76,11 @@ Docker
    container that allows custom config and will keep your settings and
    data past docker image restarts.
 
-Fedora
-~~~~~~
-
--  https://github.com/thunderbirdtr/syncthing\_rpm (SRPM, binary
-   packages coming)
-
--  https://copr.fedoraproject.org/coprs/tune2fs/syncthing/ (RPMs, based
-   on https://github.com/thunderbirdtr/syncthing\_rpm)
 
 Gentoo
 ~~~~~~
 
-Syncthing is available from the [timboudreau overlay](http://github.com/timboudreau/gentoo), or from [fearedbliss's overlay](https://github.com/fearedbliss/bliss-overlay). These overlays contain the ebuild for building syncthing from source,
-installing it, and setting it up as a service that runs on startup.
```

Comment:
Gentoo users will have to know how to handle their OS, I don't want a tutorial in here.